src/Frontend/Core/Engine/Form.php

```diff
@@ -590,7 +590,7 @@ public function getValues($excluded = array('form', 'save', '_utf8'))
      *
      * @param \SpoonTemplate $tpl The template instance wherein the form will be parsed.
      */
-    public function parse(\SpoonTemplate $tpl)
+    public function parse(SpoonTemplate $tpl)
```

Comment:
shouldn't this backslash be still here? (or a use SpoonTemplate at the top of the file)